pkg/ibmvpc/examples/out/explain_out/LBToResIPNode_all_vpcs_explain_debug.txt

```diff
@@ -0,0 +1,296 @@
+Explaining connectivity from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca to 192.168.32.5 (iks-clusterid:1[192.168.32.5]) within ky-test-vpc
+=================================================================================================================================================
+
```

Comment:
I'm not sure that in this specific case we need to split to all possible sub-subnets as per sg/nacl different rules combination. All paths are disabled since the target is not a pool member. Would the user care to see all the different rules combinations which are ill-relevant anyways?

Reply:
same answer

Reply:
all other nodeset can be suffer from the case of "one rule control them all", for example, two nodesets of the size of 4 which are not on the same vpc - you will get the exact same explain 16 times.
I dont think it is good idea to make a special implementation fo load balancer

Reply:
@kyorav @zivnevo what do you think? please review the explainability output for LB

Reply:
> I'm not sure that in this specific case we need to split to all possible sub-subnets as per sg/nacl different rules combination. All paths are disabled since the target is not a pool member. Would the user care to see all the different rules combinations which are ill-relevant anyways?

I'd not consider sg/nacl rules as necessarily irrelevant in this case, since it might be relevant to check if the connection is blocked only by LB configuration or also by nacl/sg configs.